app/components/faq/QAndAPair.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Grid } from 'semantic-ui-react';
+import './style.scss';
+
+const QAndAPair = (props) => (
```

Comment:
Instead of props you can just add `{ questionId, questionText, answerText }` so that you don't need `props.something`. Please add PropTypes.

Reply:
I passed the whole question in when using map in FaqPage.component.js, so instead using question.questionId now.